client/lib/viewport/index.js

```diff
@@ -106,18 +108,28 @@ export function isWithinBreakpoint( breakpoint ) {
 }
 
 export function addWithinBreakpointListener( breakpoint, listener ) {
```

Comment:
would you mind adding jsdoc's that explain what the type of each parameter is?

Reply:
Done.